xtrack/beam_elements/elements.py

```diff
@@ -136,6 +136,44 @@ def get_backtrack_element(self, _context=None, _buffer=None, _offset=None):
     _pkg_root.joinpath('beam_elements/elements_src/elens.h')]
 
 
+## Wire Element
+
+class Wire(BeamElement):
+    
+    _xofields={
+               'wire_L_phy'  : xo.Float64,
+               'wire_L_int'  : xo.Float64,
+               'wire_current': xo.Float64,
+               'wire_xma'    : xo.Float64,
+               'wire_yma'    : xo.Float64,
+              }
+
+    def __init__(self,  wire_L_phy   = None,
+                        wire_L_int   = None,
+                        wire_current = None,
+                        wire_xma     = None,
+                        wire_yma     = None,
+                        **kwargs):
+        super().__init__(**kwargs)
+        self.wire_L_phy   = wire_L_phy
+        self.wire_L_int   = wire_L_int
+        self.wire_current = wire_current
+        self.wire_xma     = wire_xma
+        self.wire_yma     = wire_yma
+
+    def get_backtrack_element(self, _context=None, _buffer=None, _offset=None):
+        return self.__class__(
+                              wire_L_phy   = self.wire_L_phy,
+                              wire_L_int   = self.wire_L_int,
+                              wire_current = self.wire_current,
```

Comment:
The backtrack element is meant to do the opposite action compared to the element (to be used to track particles backwards). This can be obtained by changing the sign of the current I guess.